src/types.ts

```
@@ -22,7 +22,7 @@ export type RsiProps<T extends string> = {
   // Runs after column matching and on entry change
   tableHook?: TableHook<T>
   // Function called after user finishes the flow. You can return a promise that will be awaited.
-  onSubmit: (data: Result<T>, file: File) => void | Promise<any>
+  onSubmit: (mode: string, data: Result<T>, file: File) => void | Promise<any>
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)